en/concepts/scene/shadow.md

```diff
@@ -3,35 +3,73 @@
 
 Here is an example of a shadow from __Cocos Creator 3D__:
-![shadow](shadow/shadowExample.jpg)
+![shadow](shadow/shadowExample.png)
 
 ## Turning on shadows
 
-__Cococs Creator 3D__ currently supports the highly efficient __Planar Shadow__. More __Shadow__ types will be available later. 
+__Cococs Creator 3D__ currently supports two shadow modes, __shadow Map__ and __planer Shadow__ for developers to use.
 
-It only takes two steps to enable a __Planar Shadow__ in __Cococs Creator 3D__:
+* It takes three steps to enable a __Planar Shadow__ in __Cococs Creator 3D__:
 
-  1. Select the Scene node on the level manager, you can see the following panel, check the *Enabled* property of *planarShadows*.
-  ![location of planar shadow](shadow/planarShadows.jpg)
+  1. Select the Scene node on the level manager, you can see the following panel, check the *Enabled* property of *Shadows*.
+![location of enable shadow](shadow/shadows.png)
 
-  2. Set *ShadowCastingMode* to *ON* in the model that needs to display shadows.
-  ![ShadowCastingModes property](shadow/shadowCastingMode.jpg)
+  2. Choose *Type* as *Planar*
+![location of shadow type](shadow/planarShadowType.png)
+
+  3. Set *ShadowCastingMode* to *ON* in the model that needs to display shadows.
+![ShadowCastingModes property](shadow/planarShadowCastingMode.png)
 
   > **Note**: __Planar Shadow__ will only be cast on the shadow surface. Adjusting the directional light angle can adjust the shadow projection.
 
+* It takes four steps to enable a __Shadow Map__ in __Cococs Creator 3D__:
+
+  1. Select the Scene node on the level manager, you can see the following panel, check the *Enabled* property of *Shadows*.
+![location of enable shadow](shadow/shadows.png)
+
+  2. Choose *Type* as *ShadowMap*
+![location of shadow type](shadow/shadowMapType.png)
+
+  3. Set *ShadowCasting* to *ON* in the model that needs to display shadows.
+![ShadowCastingModes property](shadow/shadowMapCastingMode.png)
+
+  4. Set *ReceiveShadow* to *ON* in the model that needs to display shadows.
+![ReceiveShadowModes property](shadow/shadowMapReceiveMode.png)
+
+  > **Note**: __ReceiveShadow__ receive the shadow effect. __ShadowCasting__ produces a shadow effect.
+
 ## PlanarShadows panel
 
-![planar shadow panel details](shadow/planarShadowsDetail.jpg)
+![planar shadow panel details](shadow/planarShadowsDetail.png)
+
+The following describes all the properties of the panel:
+
+| Properties | Explanation |
+| --- | --- |
+| **Enabled**     | Whether to turn on the shadow effect |
+| **Type**        | Choose shade type |
+| **ShadowColor** | color value of the resulting shadow |
+| **Normal**      | normals to vertical and shadow planes |
+| **Distance**    | the distance of the shadow plane from the coordinate origin in the direction of the normal |
```

Comment:
Keep the same format, upper case for the first letter

Reply:
i know